Azure-Samples/communication-services-javascript-quickstarts
# Extract teams chat thread ID from the meeting link for chat-to-teams quickstart

## Purpose

Connecting ACS chat to teams requires the teams meeting link and the chat thread ID.
The thread ID is part of the meeting link, which can be initially parsed out.
This PR enables basic parsing.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Run the code
* Paste in a teams meeting link
* Observe the chat ID populating in the input box

## Other Information
